src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -296,14 +297,18 @@ private function denormalizeRelation(string $resourceClass, string $attributeNam
     {
         if (is_string($value)) {
             try {
-                return $this->iriConverter->getItemFromIri($value);
+                return $this->iriConverter->getItemFromIri($value, true);
```

Comment:
There is still a problem. The JSON-LD serializer must not be aware of Doctrine. It's 2 100% independent components.

And this case should never happen because `getItemFromIri` will return `null` if the resource isn't found.